packages/now-node/src/index.ts

```diff
@@ -1,12 +1,17 @@
+import { fork, spawn } from 'child_process';
+import { readFileSync, lstatSync, readlinkSync, statSync } from 'fs';
+import { writeJSON, remove } from 'fs-extra';
```

Comment:
Can we do this without `fs-extra`? You could use `fs.promises` https://nodejs.org/docs/latest-v12.x/api/fs.html#fs_fs_promises_api

Reply:
Probably, but let's do it in a separate PR?

Reply:
But its relevant to this PR because you're adding the dependency. It wasn't there before 🤔